tests/rptest/services/redpanda_cloud.py

```diff
@@ -472,7 +472,11 @@ def _get_product_name(self, config_profile_name):
         return None
 
     def _create_cluster_payload(self):
-        _cidr = "10.1.0.0/16" if self.isPublicNetwork else self.config.network
+        # Previously cloud creation could be done by passing word
+        # 'private' as a _cidr. Now it is failing with HTTP;:400
+        # old code: 
+        # _cidr = "10.1.0.0/16" if self.isPublicNetwork else self.config.network
```

Comment:
This is probably better as a comment in the review than a comment in the code?

Reply:
I think it's fine to leave in that case up to you, but you could also write something independent of the old state like "we need to do it like this because X and Y".